### PR TITLE
Fix "Bad SetLocalOrigin(x,y,z) on gmod_hands" warnings

### PR DESCRIPTION
**Pull request by thecraftianman**
Closes Facepunch/garrysmod-issues#1184

This PR should fix all instances of this mildly annoying warning that can occur under normal gameplay circumstances. Hands will be (re)parented to the viewmodel only when they actually need to be, and their position will now always be set to prevent the engine from complaining when they are parented or unparented.

### CODE REVIEW

File: garrysmod/gamemodes/base/entities/entities/gmod_hands.lua [hunk 1]
```diff
@@ -46,22 +44,33 @@ function ENT:GetPlayerColor()
 
 end
 
-function ENT:ViewModelChanged( vm, old, new )
```
Comment (robotboy655) [inline garrysmod/gamemodes/base/entities/entities/gmod_hands.lua:49]: Please do not include unnecessary changes in your Pull Requests.

**Comment by thecraftianman** [comment]
Should be all good now

**Review by robotboy655 — APPROVED** [review]

**Comment by robotboy655** [comment]
Appears to be working, at least when teleporting on `gm_mobenix_v3`.

**Comment by a-straight-white-male** [comment]
Please revert, fixing an "annoying warning" is NOT worth killing half of all weapon addons!!!
